src/test/java/no/nav/tag/sykefravarsstatistikk/api/provisjonering/importering/integrasjon/utils/SykefraværsstatistikkNæringUtilsJdbcTest.java

```diff
@@ -0,0 +1,170 @@
+package no.nav.tag.sykefravarsstatistikk.api.provisjonering.importering.integrasjon.utils;
+
+import no.nav.tag.sykefravarsstatistikk.api.domene.sammenligning.Sykefraværprosent;
+import no.nav.tag.sykefravarsstatistikk.api.domene.statistikk.SykefraværsstatistikkNæring;
+import no.nav.tag.sykefravarsstatistikk.api.domene.statistikk.ÅrstallOgKvartal;
+import no.nav.tag.sykefravarsstatistikk.api.provisjonering.importering.integrasjon.CreateSykefraværsstatistikkFunction;
+import no.nav.tag.sykefravarsstatistikk.api.provisjonering.importering.integrasjon.DeleteSykefraværsstatistikkFunction;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.data.jdbc.DataJdbcTest;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@ActiveProfiles("db-test")
+@RunWith(SpringRunner.class)
+@DataJdbcTest
+public class SykefraværsstatistikkNæringUtilsJdbcTest {
+
+    @Autowired
+    private NamedParameterJdbcTemplate namedParameterJdbcTemplate;
+
+    private SykefraværsstatistikkNæringUtils utils;
+
+    private static final String LABEL = "TEST";
+
+
+    @Before
+    public void setUp() {
+        utils = new SykefraværsstatistikkNæringUtils(namedParameterJdbcTemplate);
+        cleanUpLokalTestDb(namedParameterJdbcTemplate);
+        opprettDimensjoner(namedParameterJdbcTemplate);
+    }
+
+    @After
+    public void cleanUp() {
+        cleanUpLokalTestDb(namedParameterJdbcTemplate);
+    }
+
+
+    @Test
+    public void skal_lagre_data_i_lokale_sykefraværstatistikk_tabellen(){
```

Comment:
navnekonvensjonen vi bruker på tester: `metodenavn__hva_metoden_skal_gjøre()`

Reply:
Good to know ;)